downloader/Makefile

```diff
@@ -1,4 +1,4 @@
-IMAGE := quay.io/migtools/gitops-primer-downloader
+IMAGE := quay.io/kubesaw/export-operator-downloader
```

Comment:
I don't think that we have the kubesaw quay org already set up. I can take care of it

Reply:
it looks like we need the following repos:

- snapshot-operator
- snapshot-operator-bundle
- snapshot-operator-export
- snapshot-operator-download
- snapshot-operator-webhook